snapshot-labs/sx-monorepo
# feat: make proposal sidebar resizable

### Summary



Closes: https://github.com/snapshot-labs/workflow/issues/327

This PR adds the ability to resize the proposal page right sidebar, from the default 340px up to 400px max.

### Changes

- add a new UiResizableHorizontal component, holding all the logic to resize its content
- edit the Proposal page sidebar, to wrap the sidebar inside this new component

### How to test

1. Go to any proposal page
2. Hover on the border between the main content and the right sidebar
3. You should be able to drag the border, and resize the sidebar 
4. Sidebar width can be resize from 340px to 400px
5. After resize the window, you should be able to resize 
6. On small screen, sidebar should be below the main content, and display like before
7. Refresh the page: it should remember your width setting
8. Resizing the page should keep the sidebar at the same width
